### ISSUE TITLE
Pledge to replace all SVG files with PNG or JPG

### ISSUE THREAD

**Issue by gkoumanos** [WeatherXM/docs#52]
I cannot see a reason to have graphics as SVG. Especially some files are extremely huge (17MB) and very heavy on the client (browser) to render all "paths" etc of the file.
PNG with transparency or even JPG would be a better fit IMHO.

**Comment by NikosTsiligaridis** [comment]
Agreed, makes no sense to use SVG for graphics that are not going to ever be scaled, even moreso for photos :open_mouth:
